pkg/cmd/browse/browse.go

```diff
@@ -201,8 +207,8 @@ func parseFile(opts BrowseOptions, f string) (p string, start int, end int, err
 	p = parts[0]
 	if !filepath.IsAbs(p) {
 		p = filepath.Clean(filepath.Join(opts.PathFromRepoRoot(), p))
-		// Ensure that a path using \ can be used in a URL
```

Comment:
It's not the responsibility of a function called `parseFile` to be escaping the resulting string for use in a URL later. You're on the right tracking with escaping, but it should be moved elsewhere, e.g. to just before this value gets used in a URL.